kfparser/parser_test.go

```diff
@@ -623,7 +625,7 @@ func TestParse_valid_syntax(t *testing.T) {
 			use a_ext{addr: '0x0000', seed: 3} as ext1;
 			table tt1 { tc1 int, tc2 text }
 			action act2($a) private
-			{ $v = ext1.call(2, '3', $a, @dataset, -2, 1 + - 2, (1 * 2) + 3, 1 <= 2, 1 and $a, address(@caller), address(upper(@action))); }`,
+			{ $v = ext1.call(2, '3', $a, @caller, -2, 1 + - 2, (1 * 2) + 3, 1 <= 2, 1 and $a, address(@caller), upper(@caller)); }`,
```

Comment:
Hmm, having talked about this in the meeting just now, I guess `address` *function* was removed, although for this test I think it's just arbitrary.